plugins/markdown/src/org/intellij/plugins/markdown/ui/preview/javafx/MarkdownJavaFxHtmlPanel.java

```diff
@@ -111,9 +112,12 @@ protected String prepareHtml(@NotNull String html) {
   @Override
   public void setCSS(@Nullable String inlineCss, String @NotNull ... fileUris) {
     PreviewStaticServer.getInstance().setInlineStyle(inlineCss);
-    myCssUris = inlineCss == null ? fileUris
+    PreviewStaticServer.getInstance().setStyleOverrides(StyleOverridesKt.createStyleOverrides());
```

Comment:
If you would use `object` with `@JvmStatic` we would remove from here `Kt` postfix :)